models/aeif_cond_beta_multisynapse.h

```diff
@@ -74,16 +74,18 @@
 
  neuron = nest.Create('aeif_cond_beta_multisynapse')
  nest.SetStatus(neuron, {"V_peak": 0.0, "a": 4.0, "b":80.5})
- nest.SetStatus(neuron, {'taus_decay':[50.0,20.0,20.0,20.0],
+ nest.SetStatus(neuron, {'num_of_receptors':4,
+                         'E_rev':[0.0,0.0,0.0,-85.0],
+                         'taus_decay':[50.0,20.0,20.0,20.0],
                          'taus_rise':[10.0,10.0,1.0,1.0]})
 
  spike = nest.Create('spike_generator', params = {'spike_times':
-                                                  np.array([10.0])})
+                                                 np.array([10.0])})
 
  voltmeter = nest.Create('voltmeter', 1, {'withgid': True})
 
  delays=[1.0, 300.0, 500.0, 700.0]
- w=[1.0, 1.0, 1.0, -1.0]
+ w=[1.0, 1.0, 1.0, 1.0]
  for syn in range(4):
      nest.Connect(spike, neuron, syn_spec={'model': 'static_synapse',
                                            'weight': w[syn],
```

Comment:
You mean that the order should be important? I do not know, I did not see any difference in the simulations if I put 'receptor_type' at the end of the syn_spec list, in any case now I've put it in the second position after 'model'.